index.bs

```diff
@@ -615,7 +615,7 @@ when <a href="https://wicg.github.io/fenced-frame/#issue-c137ca33">the fetch int
 
 # Permissions Policy integration # {#permission-policy-integration}
 
-This specification defines a [=policy-controlled feature=] identified by the string "<code><dfn noexport>attribution-reporting</dfn></code>".
+This specification defines a [=policy-controlled feature=] identified by the string "<code><dfn>attribution-reporting</dfn></code>".
```

Comment:
We may want to use a more specific ID for this instead of just `attribution-reporting`.

Reply:
Added "export for=attributionreporting", or do you have any suggestion? Thanks.

Reply:
I was thinking something that would cause the URL to be https://wicg.github.io/attribution-reporting-api/#permissions-policy instead of https://wicg.github.io/attribution-reporting-api/#attribution-reporting.

Reply:
Hmmm I can' find a good example that exports a permission policy. Are you aware of any?

Reply:
I am not.